src/test/cli/rbd/help.t

```diff
@@ -110,6 +113,40 @@
     --io-total arg       total size to write (in B/K/M/G/T)
     --io-pattern arg     write pattern (rand or seq)
   
+  rbd help cg create
+  usage: rbd cg create [--pool <pool>] 
+                       <cg-name> 
```

Comment:
Frankly, I find the current design of cli ambiguous. What is the benefit of being able to specify group name as a positional parameter as well as a named parameter as well as pool using pool/image or using a named parameter? I definitely will implement it the way you want, just wondering.